homework/01LessonOne.js

```diff
@@ -144,7 +146,7 @@ var notNotUndefined;
 // Explain truthiness and falsiness in your own words. Provide an example for each.
 //
 // Your Answer Goes Here:
-//
+// Truthy means that a value coerces to true when evaluated in boolean context, while falsy eans that a value coerces to false when evaluated in boolean context.
```

Comment:
close - just remember that `==` performs additional actions on the two sides being evaluated in order to produce `true` or `false`